README.md

```diff
@@ -1 +1,10 @@
-https://codeday.org
\ No newline at end of file
+<p align="center">
+  <br><br>
+  <img width="500px" src="https://github.com/giridhar7632/www/blob/patch-1/public/assets/img/logo-text-color.png?raw=true" alt="codeday"><br>
```

Comment:
Could you link this using a relative link, instead of an absolute link to your repository? Otherwise the image will break whenever your fork is deleted.